presto-main/src/main/java/io/prestosql/sql/planner/planprinter/IoPlanPrinter.java

```diff
@@ -244,36 +247,36 @@ public boolean equals(Object obj)
                 }
                 TableColumnInfo o = (TableColumnInfo) obj;
                 return Objects.equals(table, o.table) &&
-                        Objects.equals(columnConstraints, o.columnConstraints) &&
+                        Objects.equals(columns, o.columns) &&
                         Objects.equals(estimate, o.estimate);
             }
 
             @Override
             public int hashCode()
             {
-                return Objects.hash(table, columnConstraints, estimate);
+                return Objects.hash(table, columns, estimate);
             }
 
             @Override
             public String toString()
             {
                 return toStringHelper(this)
                         .add("table", table)
-                        .add("columnConstraints", columnConstraints)
+                        .add("columns", columns)
                         .add("estimate", estimate)
                         .toString();
             }
         }
     }
 
-    public static class ColumnConstraint
+    public static class Column
```

Comment:
I don't see a good reason for changing the name of this class, so I'd keep it as it was before. At a minimum, if we're changing it to make usage or meaning clearer, it should be done in a separate commit.